.github/workflows/ci.yml

```diff
@@ -1,26 +1,77 @@
 name: CI
 
-on: [push]
+on: [push, pull_request]
```

Comment:
They aren't redundant for outside contributions, where the pull request branch is pushed to a fork, since github actions aren't automatically enabled on forks.

Reply:
Maybe we could ignore the event in the case where they are redundant.

Reply:
Added the conditional to skip the job if it is for a pull request event for a branch that is on this repo.  You can see that on the checks for this PR, which includes "CI / build (pull_request) Skipped".  I created a fork and opened a PR from that fork to test that scenario (https://github.com/Shopify/identity_cache/pull/480) which used to only show the CLA check with `on: [push]` and now runs CI when the PR is opened or when a push is made to the PR branch.